.ci/integration-tests

```diff
@@ -56,18 +90,23 @@ function setup_environment() {
         printf "Go build Failure\n"
         terminate_script
     fi
+}
 
+# run_controller runs MCM controller in background
+# it expects the port to be passed as a parameter
+function run_controller() {
     printf "\nRunning MCM in background\n"
     ./controller_manager \
-                --control-kubeconfig=$control_kubeconfig \
-                --target-kubeconfig=$target_kubeconfig \
-                --namespace=$control_namespace \
-                --safety-up=2 \
-                --safety-down=1 \
-                --machine-drain-timeout=5 \
-                --machine-health-timeout=10 \
-                --machine-set-scale-timeout=20 \
-                --v=2 > mcm.out 2>&1 &
+        --control-kubeconfig=dev/control_kubeconfig_${provider}.yaml \
+        --target-kubeconfig=dev/control_kubeconfig_${provider}.yaml \
```

Comment:
Should this be target_kubeconfig_${provider}.yaml ?

Reply:
Yes, you are right. I am wondering how the integration tests succeeded then ❓